DateTimePicker/DateTimePicker.js

```diff
@@ -10,35 +10,15 @@
  */
 
 import React from 'react';
```

Comment:
I think the order of this line would be the next line to `import {Cell, Row} from '@enact/ui/Layout';`